constants/style.js

```diff
@@ -0,0 +1,9 @@
+export default {
+  marginHorizontal: 20,
```

Comment:
the general margin values should be here like

```
margin: {
  small: 8,
  large: 10,
}
```
Also, 20 is not a standard margin value.